packages/devtools_app/lib/src/shared/inspector_service.dart

```diff
@@ -98,9 +98,7 @@ abstract class InspectorServiceBase extends DisposableController
   /// Daemon API calls won't execute until after the current frame is done
   /// rendering.
   bool get useDaemonApi {
```

Comment:
super nit, this could be a one liner now that it fits on one line: `bool get useDaemonApi => !serviceManager.isMainIsolatePaused;`